hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -161,61 +168,62 @@ This is intended to be unique but also provide structured information to the DHC
+Version: 2020-04-16  
 
 #### GATEWAY
-The gateway address to use if the TFTP server is on a differenet subnet e.g. "192.168.0.1"
+The gateway address to use if the TFTP server is on a differenet subnet e.g. "192.168.0.1"  
+
 Default: ""  
-Version: pieeprom-2020-04-16.bin  
+Version: 2020-04-16  
 
 #### MAC_ADDRESS
 Overrides the Ethernet MAC address with the given value. e.g. dc:a6:32:01:36:c2  
+
 Default: ""   
-Version: pieeprom-2020-04-16.bin
+Version: 2020-04-16  
 
 ### DISABLE_HDMI
 The [HDMI boot diagnostics](./boot_diagnostics.md) display is disabled if DISABLE_HDMI=1. Other non-zero values are reserved for future use.
 
 Default: 0   
-Version: pieeprom-2020-04-16.bin  
+Version: 2020-04-16  
 
-`Version: pieeprom-2020-06-15.bin - STABLE`
-The `disable_splash` property is no longer checked because the HDMI diagnostics screen is started before config.txt is read.
+From version 2020-09-03 the `disable_splash` property in `config.txt` is no longer checked because the HDMI diagnostics screen is started before `config.txt` is read.
 
 ### ENABLE_SELF_UPDATE
-Allows the bootloader to update itself instead of requiring recovery.bin. This is intended to make it easier to update the bootloader firmware via network boot. To enable set `ENABLE_SELF_UPDATE=1` and add `bootloader_update=1` in config.txt.
-N.B. There is no automatic rollback in the event of a power failure during the firmware update. In the unlikely event of this happening you will have to use Pi Imager to apply the rescue image.
-
-If self update is enabled then the bootloader will look for (pieeprom.upd + pieeprom.sig) and/or (vl805.bin + vl805.sig) on the boot partition (or TFTP root). If the update files are different to the current image then the update is applied and system is reset. Otherwise, if the images are identical then boot continues as normal.
+Enables the bootloader to update itself from a network or USB-MSD boot.
 
-Since the updates files are in the same format as generated by rpi-eeprom-update you can use rpi-eeprom-update to install the files to /boot so long as /boot mounts the approprate boot device / tftp-root (via NFS).
+If self update is enabled then the bootloader will look for (pieeprom.upd + pieeprom.sig) and/or (vl805.bin + vl805.sig) on the USB MSD boot partition (or TFTP root). If the update image differs from the current image then the update is applied and system is reset. Otherwise, if the EEPROM images are byte-for-byte identical then boot continues as normal.
 
-**The bootloader only reads the configuration from the bootconf.txt in the EEPROM and not the one in the update files. Therefore, in order to enable `ENABLE_SELF_UPDATE` you have to first update the bootloader via the SD-CARD or FLASHROM.**
+Since the updates files are in the same format as generated by rpi-eeprom-update you can use rpi-eeprom-update to install the files to `/boot` so long as `/boot` mounts the approprate boot device / tftp-root (via NFS).
```

Comment:
I'm not sure what this paragraph is saying. Is it saying that you can run `rpi-eeprom-update` to put the files required by self update into the boot filesystem? If so, you probably want to be more explicit. Also, am I right in saying that the bootloader will do a self-update after the rpi-eeprom-update service is run automatically, whenever the next boot happens? If you're mentioning the manual method, you should probably mention this automatic method too.

Perhaps something like this:

`The update files used by the self update feature can be created in two ways: by the rpi-eeprom-update service, or by manually running the rpi-eeprom-update command.`

With rpi-eeprom-update in backticks.

(That also has the advantage that it is shorter - less for folk to take in).

Reply:
removed it  - it just says that you can use rpi-eeprom-update